### DISCUSSION TITLE
Single cable solution

### DISCUSSION THREAD

**Discussion by cawv** [fabianishere/udm-iptv#160] [Q&A]
Hi, just passing by to check if this solution can provide both IPTV and Internet simultaneously.
My home setup have only one Cat6 cable to each room, must have a "trunked" Vlan's on the same cable.
Does this solution provide this?

thanks!

**Comment by fabianishere** [comment]
I don’t know how IPTV is routed on your network, but you can use a managed switch (like the USW Flex Mini) to untag the IPTV traffic in each room.

**Reply by cawv** [reply]
Hi, my current setup is:

Vivo FTTH Fiber ---->   ONU/HGU Askey 3507 -----> Linksys Gigabit unmanaged switch ----->  Vivo STB's / Linksys Velop AP    (each room have one STB and one AP for cabled and wifi access)

As you can see its pretty simple but we are having net constrains most of the time on the internet traffic and lost connections, probably due to the routing rellying on the old and poor Askey Router and maybe from the TV traffic flooding the local network.

Vivo uses Vlan 10 for Internet, Vlan 20 for IPTV and Vlan 30 for telphone.
They use the old Mediaroom system much like ATT Uverse.

Any help on how to use my UDM Pro to route the traffic for at least the Internet and TV will be nice, as long as they are on the same cable. We can opt for a switch at the end to untag, but getting rid of the Askey for the internet routing is the goal.

regards.

**Reply by fabianishere** [reply]
If you replace the Askey with the UDM Pro, you can use this script to route the IPTV traffic onto one of your LANs.

The best would be to put the IPTV traffic on a separate VLAN, but that would require untagging the packets in each room.

If you install the script on your UDM Pro, you can select the Vivo profile and I hope it will work out of the box then.

**Reply by cawv** [reply]
If I keep the Askey for IPTV I can bridge it only the internet Vlan.
The UDM can do the internet routing and passthrough the IPTV if I join the internet and IPTV traffic in the UDM and then untag at each room?

**Reply by fabianishere** [reply]
Hmm I don’t exactly understand your question. Do you want to keep the Askey?

The UDM can receive both the internet and IPTV traffic and route it to your LAN or any tagged VLANs, so you do not need the Askey.

**Reply by cawv** [reply]
I was looking for the simplest solution.
I'm new to this networking world and just got a UDM recentrly trying to improve my home internet and IPTV.
Saw some articles about IPTV on UDM but all of them involve some deep knowledge and comand lines.
Hope there is a simpler solution.
I dont mind keeping the Askey at the end, as long as it doesnt handle the internet as it is my network bottleneck.

regards,

**Reply by fabianishere** [reply]
If you want to use the UDM, then you'll need to follow the guide of this project. I don't think there is a simpler solution right now.

**Reply by Edu-ST** [reply]
> If you replace the Askey with the UDM Pro, you can use this script to route the IPTV traffic onto one of your LANs.
> 
> The best would be to put the IPTV traffic on a separate VLAN, but that would require untagging the packets in each room.
> 
> If you install the script on your UDM Pro, you can select the Vivo profile and I hope it will work out of the box then.

@fabianishere   I did all exactly all steps installing the script and selecting Vivo option in UDM Pro.
It connected but keep up  few seconds and freeze...
If I change the channel on TV box, it connect again for few seconds more and freeze...
Do you think anything to fix it?
Thanks

**Reply by fabianishere** [reply]
In that case, IGMP Proxy is probably not working. Could you share the diagnostic details in the discussion you opened?

**Reply by Edu-ST** [reply]
> In that case, IGMP Proxy is probably not working. Could you share the diagnostic details in the discussion you opened?

Sure..   which should be better to get  look?
I would like "to pay a beer" to help us    👍
